<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1264</width>
    <height>902</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1121</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1121</width>
       <height>566</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Milestone1 - Page 1</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>561</width>
        <height>211</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(136, 194, 211);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="title">
       <string>Select Location</string>
      </property>
      <widget class="QComboBox" name="stateList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
      </widget>
      <widget class="QLabel" name="stateLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>State</string>
       </property>
      </widget>
      <widget class="QLabel" name="cityLabel">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>20</y>
         <width>47</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>City</string>
       </property>
      </widget>
      <widget class="QListWidget" name="cityList">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>50</y>
         <width>171</width>
         <height>151</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
      </widget>
      <widget class="QLabel" name="zipCodeLabel">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>20</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Zip Code</string>
       </property>
      </widget>
      <widget class="QListWidget" name="zipCodeList">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>50</y>
         <width>171</width>
         <height>151</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="businessGroup">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>881</width>
        <height>341</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(136, 194, 211);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="title">
       <string>Businesss</string>
      </property>
      <widget class="QTableWidget" name="businessTable">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>821</width>
         <height>281</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>true</bool>
       </attribute>
      </widget>
     </widget>
     <widget class="QGroupBox" name="ZipcodeStatistics">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>0</y>
        <width>321</width>
        <height>211</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(136, 194, 211);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="title">
       <string>Zipcode Statistics</string>
      </property>
      <widget class="QLineEdit" name="NumOfBusinessesBox">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>30</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="NumOfBusinessesLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Number of Businesses:</string>
       </property>
      </widget>
      <widget class="QLabel" name="TotalPopLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Total Population:</string>
       </property>
      </widget>
      <widget class="QLabel" name="AvgIncomeLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Average Income:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="TotalPopBox">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>90</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="AvgIncomeBox">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>150</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1264</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
